### ISSUE TITLE
Predict only 1 image using the Prototypical Network

### ISSUE THREAD

**Issue by pices45-cyber** [sicara/easy-few-shot-learning#144]
**Problem**
How to use the model in evaluating only 1 picutre?

**Considered solutions**
I tried to load only 1 image.

**How can we help**
Provide specific function/code for evaluating/predicting 1 picture

**Comment by ebennequin** [comment]
Can you please add details on what you tried? At the very least please share your code, its expected result and its actual result.

**Comment by pices45-cyber** [comment]
The code is the same. However, I want to deploy the protonet model into a mobile application. I want to know what fucnction that I need to use let the model predict only 1 image. To make it simplier, I just need a "predict" function just like other classical algorithm that I can use to evaluate only 1 image. For example, I have a FSL model that has been trained and evaluated, then I want it to predict the label of a single picture. How can I do that? Is there a function like model.predict("image")? 

Context: I am using FSL (ProtoNet) to classify rice diseases and I gathered primary images of the diseases. Currently, I am done with loading the data and using the pretrained model (resnet) in evaluating the task images. Now, I the only thing I want to know is how to make the model predict only 1 query.

**Comment by ebennequin** [comment]
This seems to be the same issue as #17. I suggest you look into the response here. I'm sorry I can't help you more precisely than that if you won't share the details I asked for 😶

**Comment by pices45-cyber** [comment]
Hi! I read the thread of issue #17 and I think we have the same problem. Thank you for the response 😊.

**Comment by pices45-cyber** [comment]
Ah btw I have last question. How can I save the progress of my model when its training. I only use free subscription on google colab. I want to save my training progress then resume it for other day.

**Comment by deemoe404** [comment]
> Ah btw I have last question. How can I save the progress of my model when its training. I only use free subscription on google colab. I want to save my training progress then resume it for other day.

I believe `checkpoint` in Torch is exact what you need.🤣
